jena-fuseki2/jena-fuseki-ui/src/views/dataset/Query.vue

```diff
@@ -212,8 +212,8 @@ export default {
         this.onYasqeOptionsChange()
         this.syncYasqePrefixes()
         this.loading = false
-      })
-    }, 300)
+      }, 300)
+    })
```

Comment:
:lady_beetle: it worked because JS accepts the extra parameter in the wrong function, and then appears to just use `0` (like a `this.wait()` in Java I think).